src/git-sync-all

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helpers/helpers.sh"
+
+
+function preconditions {
+  fetch
+}
+
+
+function skip_message {
+  echo "To skip the sync of the '$(get_current_branch_name)' branch"
+}
+
+
+function skippable {
```

Comment:
This is more generic. skippable is a function used by the command list architecture and is just used to determine whether or not to print the a message saying you can run `--skip`. I will add a comment to the placeholder function in script_helpers.